### ISSUE TITLE
No good gallery example for Region Adjacency Graphs

### ISSUE THREAD

**Issue by stefanv** [scikit-image/scikit-image#2589]
There were some neat blog posts during the GSoC, let's bring them into the gallery in some form.

**Comment by sciunto** [comment]
@stefanv Do you have pointers? Alternatively, I made an example for my skimage lecture that I can share as well.

**Comment by emmanuelle** [comment]
ping @stefanv this might be a good thing to do during the SciPy sprint

**Comment by PeterJackNaylor** [comment]
In the galery they are several notebooks about Region Adjacency Graphs. 
In particular: http://scikit-image.org/docs/dev/auto_examples/segmentation/plot_rag_merge.html#sphx-glr-auto-examples-segmentation-plot-rag-merge-py

Should this issue be closed?

**Comment by stefanv** [comment]
This issue refuted being closed for a reason ;)

**Comment by stefanv** [comment]
http://melvincabatuan.github.io/RAG/

**Comment by stefanv** [comment]
https://vcansimplify.wordpress.com/tag/gsoc/

**Comment by jni** [comment]
@stefanv LOL I totally forgot to close it after that. But anyway, what specifically in those blogs do you think is missing from our gallery? We have examples of everything I can see in there I think...

**Comment by stefanv** [comment]
Do we have an example of graph simplification (region merging)?

**Comment by jni** [comment]
http://scikit-image.org/docs/dev/auto_examples/segmentation/plot_rag_merge.html#sphx-glr-auto-examples-segmentation-plot-rag-merge-py
http://scikit-image.org/docs/dev/auto_examples/segmentation/plot_boundary_merge.html#sphx-glr-auto-examples-segmentation-plot-boundary-merge-py
http://scikit-image.org/docs/dev/auto_examples/segmentation/plot_rag_mean_color.html#sphx-glr-auto-examples-segmentation-plot-rag-mean-color-py

**Comment by jni** [comment]
@stefanv that's 3 of **7** RAG examples. I don't think there is a single other feature of scikit-image with better coverage in the gallery. :joy:

**Comment by stefanv** [comment]
OK, so what caught me was the naming: unless you know it's called RAG, searching for "adjacency" or "graph" doesn't get you to those examples.  OK if we write out the titles explicitly?

**Comment by jni** [comment]
oh yeah, that's totally a good idea. =) You should have both spellings in the title maybe?

**Comment by jni** [comment]
Yes, that's the gist @anamfatima1304 and if you make that PR we could close a >6 year old issue! 😂 👏 

One small suggestion: I would put the abbreviation in parentheses, not the full title. So: "Region adjacency graph (RAG) thresholding".

**Comment by jni** [comment]
🚀
